README.md

```diff
@@ -18,2309 +18,91 @@ The Ethereum connector allows you to access the Ethereum JSON RPC API through Ba
-  - `gas`: gas provided by the sender.
-  - `input`: the data send along with the transaction.
-
-
-##### Example
-
-```ballerina
-//code
-json response = {};
-http:HttpConnectorError e;
-response, e = ethereumConnector.ethGetTransactionByHash("2.0", 1999, "0x1c80b47e4a28c7c4892460c84a1f7b10aa2a33f0059f881abbcf77ef7ae2914a");
-if (e == null) {
-    println(response);
-} else {
-    println(e);
-}
-```
-
-```json
-{
-  "id":1999,
-  "jsonrpc":"2.0",
-  "result": {
-    "blockHash":"0x8655b1b7949f45a95a77c8ff3942116db22ce268b5f63d2c25f09b5625b02607",
-    "blockNumber":"0x58b",
-    "from":"0x0eb8a07d29f5afdcbec1a9d087ece456139bfb87",
-    "gas":"0x15f90",
-    "gasPrice":"0x430e23400",
-    "hash":"0x1c80b47e4a28c7c4892460c84a1f7b10aa2a33f0059f881abbcf77ef7ae2914a",
-    "input":"0x725b107db88f11690dd6f2032f260a0a93d735bd15e503d71bdbc1d7e73141a44bbeab080014866506b1e64281fc20690068a02f95298a09dd878e2272ca70b01c",
-    "nonce":"0x19",
-    "to":"0xee76e1d9ad8859ac9340b07e6901a028a1101577",
-    "transactionIndex":"0x0",
-    "value":"0xa",
-    "v":"0x42",
-    "r":"0x7439005bf1616c79a13ccb98b9020d791a801436e16b69ae3bdf04954760e1e3",
-    "s":"0x7b400bc21ca9649511f681bc46be18d7cd0ddc74d03b1095ef78bb4231fbcded"
-    }
-}
-```
-
-##### Quick Testing
-`bin$ ballerina run sample.bal "http://localhost:8080" "2.0" 1999 eth_getTransactionByHash  0xf6c17bf68c909565f2f1766da00a0f44e52ecb381ac6c8e088d28273d92e79ef`
-
-##### Related Ethereum Documentation
-
-https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_gettransactionbyhash
-
-***
-
-#### ethGetTransactionByBlockHashAndIndex
-
-Returns information about a transaction by block hash and transaction index position.
-
-
-##### Parameters
-
-1. `string` - JSON RPC version
-2. `int` - network id
-3. `string`, 32 Bytes - hash of a block.
-4. `string` - integer of the transaction index position.
-
-##### Returns
-
-See [ethGetBlockByHash](#ethgettransactionbyhash)
-
-##### Quick Testing 
-`bin$ ballerina run sample.bal "http://localhost:8080" "2.0" 1999 eth_getTransactionByBlockHashAndIndex  0xf6c17bf68c909565f2f1766da00a0f44e52ecb381ac6c8e088d28273d92e79ef 0x0`
-
-##### Related Ethereum Documentation
-
-https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_gettransactionbyblockhashandindex
-
-***
-
-#### ethGetTransactionByBlockNumberAndIndex
-
-Returns information about a transaction by block number and transaction index position.
-
-
-##### Parameters
-
-1. `string` - JSON RPC version
-2. `int` - network id
-3. `string` - a block number, or the string `"earliest"`, `"latest"` or `"pending"`.
-4. `string` - the transaction index position.
-
-
-##### Returns
-
-See [ethGettransactionbyhash](#ethgettransactionbyhash)
-
-##### Quick Testing
-`bin$ ballerina run sample.bal "http://localhost:8080" "2.0" 1999 eth_getTransactionByBlockNumberAndIndex  0xaf 0x0`
-
-##### Related Ethereum Documentation
-
-https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_gettransactionbyblocknumberandindex
-
-***
+### Compatibility
```

Comment:
Do we need to say Compatibility here? This should have the same format as all other packages, e.g., https://central.ballerina.io/wso2/twitter